Documentation/network/servicemesh/envoy-load-balancing.rst

```diff
@@ -0,0 +1,116 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_envoy_load_balancing:
+
+********************************************
+Proxy Load Balancing for Kubernetes Services
+********************************************
+
+This guide explains how to configure Proxy Load Balancing for Kubernetes
+services using Cilium, which is useful for some use cases such as gRPC
+load-balancing.
+
```

Comment:
random question: is this a beta feature? Does it need the beta include?

Reply:
This is good point 💯, we have one beta template to include (e.g. beta.rst).